### ISSUE TITLE
Video streaming error

### ISSUE THREAD

**Issue by singhayushh** [ProjectSakura/Mirai#11]
### Issue Type
Bug Report

### About the issue
Hello, I am using Firefox in Kubuntu (LTS 20.04.2). Whenever I'm trying to play any of the episodes, I am getting the following error:

![image](https://user-images.githubusercontent.com/52620158/111222223-8c305880-8601-11eb-88f8-52075c9aedb9.png)

I have tried running the same on Chrome but the error still persists.
> This issue can be taken by any willing contributor or participant of GSSoC!

**Comment by ArunTeltia** [comment]
Yeah This is bound to happen because we dont have the high access servers, the website is currently hosted on heroku and the data we are scraping takes some time, If we host this website on one of the cloud services, this may get resolved i am not sure tho. It needs the research, 

PS: I am not tagging any issue with tags because the Pr needs to be tagged as gssoc21 so that one can redeem the points, Please have a look if your points are added to your profile or now
